File: app/src/main/java/my/banancheg/com/weather_hw6/entity/Sys.java

package my.banancheg.com.weather_hw6.entity;

import java.io.Serializable;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import org.apache.commons.lang3.builder.ToStringBuilder;

public class Sys implements Serializable
{

    @SerializedName("pod")
    @Expose
    private String pod;
    private final static long serialVersionUID = 2600681293992773302L;

    public String getPod() {
        return pod;
    }

    public void setPod(String pod) {
        this.pod = pod;
    }

    public Sys withPod(String pod) {
        this.pod = pod;
        return this;
    }

    @Override
    public String toString() {
        return new ToStringBuilder(this).append("pod", pod).toString();
    }

}

File: app/src/main/java/my/banancheg/com/weather_hw6/entity/Sys.kt
package my.banancheg.com.weather_hw6.entity

import java.io.Serializable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import org.apache.commons.lang3.builder.ToStringBuilder

class Sys : Serializable {

    @SerializedName("pod")
    @Expose
    var pod: String? = null

    override fun toString(): String {
        return ToStringBuilder(this).append("pod", pod).toString()
    }

    companion object {
        private const val serialVersionUID = -6523766901800079623L
    }

}
